vizro-core/tests/unit/vizro/models/test_page.py

```diff
@@ -110,7 +110,10 @@ def test_valid_component_types(self, standard_px_chart, standard_dash_table):
         [vm.Checklist(), vm.Dropdown(), vm.RadioItems(), vm.RangeSlider(), vm.Slider()],
     )
     def test_invalid_component_types(self, test_component):
-        with pytest.raises(ValidationError, match=re.escape("(allowed values: 'button', 'card', 'graph', 'table')")):
+        with pytest.raises(
+            ValidationError,
+            match=re.escape("(allowed values: 'button', 'card', 'graph', 'table', 'container')"),
```

Comment:
```suggestion
            match=re.escape("allowed values: "),
```

Reply:
I think we've had this discussion one time and said that we want to explicitly write out the allowed values here. Even though this test has to be updated every time we add a new component, I still prefer to have this test more verbose and explicit about which values are allowed.

Reply:
Actually, never mind - in the test above we actually test for valid components and list them out explicitly already. So this one here is a bit redundant. I'll remove it!

Reply:
If I understand, you mean:
- we kill the `test_valid_component_types` test
- list the valid components in the `match`?

Clever idea! But I think we miss then the case in which a valid component doesn't work for some reason, which could happen if someone creates a new component, adds it to the union, but say forgets to adhere to something. Maybe not so likely, but the test is already written lol

Reply:
But `test_valid_component_types` also wouldn't capture that 😄  And I think we do test these use cases actually in the types tests on custom components that Antony wrote?

Reply:
I thought we were testing the instatiation of the model with various valid and invalid components. In my eyes it would suffice even to just check if one invalid component fails, without a specific error message, as this really is testing pydantic.

But maybe I also don't get smth, happy either way

Reply:
Yes, that's what I also just thought. I think we can:
- Kill `test_valid_component_types`
- Test only one invalid input e.g. `vm.Dropdown`
- Keep the error message though with the valid inputs

I'll push that change then 👍